### PR TITLE
Fix Utilities.host/Utilities.platform

### LINKED ISSUE #133: IsAddin check fails in Browser, still returns true and uses Dialog framework
In a browser the Utilities.isAddin check return true, even when using it outside of Office. This results in the helpers trying to use the Dialog framework to open an authentication window instead of a normal browser popup.

Problematic code:
https://github.com/OfficeDev/office-js-helpers/blob/a7b02821b9ffc9a5d89efaaa2d8f8de701524dc2/src/helpers/utilities.ts#L180

When used in the browser, the `host` property is undefined. This doesn't equal WEB and therefor it is considered being in an AddIn.

Not sure who owns this anymore, so I'm tagging all: @casieber, @Zlatkovsky, @sumurthy & @LouMM

### PR DESCRIPTION
**Pull request by Zlatkovsky**
Before this fix, "host" would return "undefined" instead of web.
This also fixes #133

### CODE REVIEW

**Comment by Zlatkovsky** [comment]
@sumurthy , @akrantz , @LouMM :  could you please review, merge in, and release a new package?
This addresses both the customer-reported issue #133 from this morning, and also a longstanding issue I'd seen in Script Lab (which is why we were stuck on an old version of 0.7.4 rather than upgrading to the latest).

Thanks!

File: src/helpers/utilities.ts [hunk 1]
```diff
@@ -45,7 +45,13 @@ function getHostInfo(): {
   // when queried from within an add-in.
   // If the platform already exposes that info, then just return it
   // (but only after massaging it to fit the return types expected by this function)
-  const context: IContext = ((window as any).Office && (window as any).Office.context) || useHostInfoFallbackLogic();
+  const isHostExposedNatively =
```
Comment (akrantz) [inline src/helpers/utilities.ts]: Why do you repeatedly do the casting? It appears this wants to check if window.Office.context.host is defined, and call useHostInfoFallbackLogic() if not. Is this fix here that it checks for the Office.context.host property and not just whether Office.context is defined?

Comment (Zlatkovsky) [reply src/helpers/utilities.ts]: Just pushed a change that addresses this.

**Comment by Zlatkovsky** [comment]
@akrantz , I see you merged the branch into master.  Does that kick off an npm publish automatially?  Or if not, can you or @sumurthy do one manually?  (Note, you may also want to merge in https://github.com/OfficeDev/office-js-helpers/pull/125 at the same time, which is another trivial PR, and publish an update that contains both -- I know Yutao has been waiting on it for several months now...)

**Comment by akrantz** [comment]
@Zlatkovsky I actually don't know. I don't normally work in this repository.

**Pull request merged**